src/api/operator/utils.ts

```diff
@@ -49,3 +48,100 @@ export function getOrderStatus(order: RawOrder): OrderStatus {
     return 'open'
   }
 }
+
+/**
+ * Get order filled amount, both as raw amount (in atoms) and as percentage (from 0 to 1)
+ *
+ * @param order The order
+ */
+export function getOrderFilledAmount(order: RawOrder): { amount: BigNumber; percentage: BigNumber } {
+  let executedAmount, totalAmount
+
+  if (order.kind === 'buy') {
```

Comment:
dont' we need to add the fee to the selling amount when it's a buy?

Reply:
On both order types (sell & buy) the fee must be taken into account, but only for executed sell amounts.
That's because we only take the fee in the sell token.
So, executed buy amounts are indifferent to the fee